src/index.html

```diff
@@ -23,10 +24,43 @@
 
     </head>
     <body class="flex-container" x-data="carFestival()">
-        <div class="excuse-top">I need a new car because ...</div>
+        <div class="excuse-top"><h1>I need a new car because ...</h1></div>
         <div class="excuse-bottom" x-show="!showGuessButton" x-text="currentExcuse"></div>
         <div class="generate-button-container" x-show="showGuessButton">
-            <button type="button" class="generate-button" x-on:click="guess">Generate excuse</button>
+            <button type="modalbutton" class="generate-button" x-on:click="guess">Generate excuse</button>
```

Comment:
You changed the existing button's type to "modalbutton" ... that doesn't seem HTML compliant ;)

Reply:
Html...schmahtml